.github/workflows/test-security-hub-scan.yaml

```diff
@@ -0,0 +1,47 @@
+name: Test security-hub-scan
+
+on:
+  workflow_call: ~
+
+
+jobs:
+  run-test:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      id-token: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+      # run setup to get uds to use zarf
+      - name: Environment setup
+        uses: ./.github/actions/setup
+
+      - name: build example package
+        id: setup
+        run: |
+          uds run create-nginx-package --no-progress
```

Comment:
we should not rely on app-specific tasks in callable workflows (other apps should be able to use this flow as is without needing to add an extra `create-nginx-package` task)

Reply:
I have changed it to a pull request triggered workflow, so it should be fine to use this task, correct?

Reply:
That should be fine for now - eventually we will fix Maru and then this can be callable again with a reference to `create:package`

Reply:
@Racer159 Ok, I think I have this resolved per your comments. Can you take a look?